x-pack/plugins/security_solution/public/timelines/components/flyout/add_timeline_button/index.test.tsx

```diff
@@ -67,24 +80,18 @@ describe.skip('AddTimelineButton', () => {
     });
 
     test('it renders create timeline btn', async () => {
-      await waitFor(() => {
-        wrapper.find('[data-test-subj="settings-plus-in-circle"]').last().simulate('click');
-        expect(wrapper.find('[data-test-subj="create-default-btn"]').exists()).toBeTruthy();
-      });
+      wrapper.find('[data-test-subj="settings-plus-in-circle"]').last().simulate('click').update();
```

Comment:
Could we try putting back `waitFor` and just moving `...simulate(click)` out of waitFor? If that works fine, let's apply this to rest of the tests in this file.